visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1975,6 +1975,65 @@ namespace pcl
       protected:
         /** \brief The render window interactor. */
         vtkSmartPointer<vtkRenderWindowInteractor> interactor_;
+		
+		 /** \brief Removes a vtk actor from the screen.
+          * \param[in] actor a pointer to the vtk actor object
+          * \param[in] viewport the view port where the actor should be removed from (default: all)
+          */
+        bool
+		removeCustomActorFromRenderer(const vtkSmartPointer<vtkLODActor> &actor,
+                                 int viewport = 0)
+		{
+			removeActorFromRenderer (actor,viewport);
```

Comment:
Call `removeCustomActorFromRenderer` from `removeActorFromRenderer` so you don't get the warnings for using deprecated functions.

Also replace the call-sites of the newly deprecated function to use the newer function instead

Reply:
I did the changes. Please do verify it and let me know if I did any mistakes. Thanks